imports/ofx/datetime.go

```diff
@@ -0,0 +1,45 @@
+package ofx
+
+import (
+	"database/sql/driver"
+	"fmt"
+	"github.com/aclindsa/ofxgo"
+	"github.com/aclindsa/xml"
+	"github.com/mattn/go-sqlite3"
+	"time"
+)
+
+// 20210310002816.000[+3:MSK]
+// 20210318
```

Comment:
Don't quite get what these comments mean 😬

Reply:
Ok, I got it. It's two timestamp formats from the OFX file. How's about getting rid of `ofxgo` lib just for parsing datetimes and do it by ourselves? Seems reasonable for me to not include the whole library just for the datetime parsing.

Reply:
Yup, my bad, forgot to delete the comments.

I thought of getting rid of `ofxgo`, but decided that copy-pasting this function is worse than an additional dependency: https://github.com/aclindsa/ofxgo/blob/3240ef383b1eea732315a3034c78e69cf4f2d886/types.go#L106